src/renderer/components/+landing-page/workspace-cluster-menu.tsx

```diff
@@ -0,0 +1,74 @@
+import React from "react";
+import { ClusterItem, WorkspaceClusterStore } from "./workspace-cluster.store";
+import { autobind, cssNames } from "../../utils";
+import { MenuActions, MenuActionsProps } from "../menu/menu-actions";
+import { MenuItem } from "../menu";
+import { Icon } from "../icon";
+import { Workspace } from "../../../common/workspace-store";
+import { clusterSettingsURL } from "../+cluster-settings";
+import { navigate } from "../../navigation";
+
+interface Props extends MenuActionsProps {
+  clusterItem: ClusterItem;
+  workspace: Workspace;
+  workspaceClusterStore: WorkspaceClusterStore;
+}
+
+export class WorkspaceClusterMenu extends React.Component<Props> {
+
+  @autobind()
+  remove() {
+    const { clusterItem, workspaceClusterStore } = this.props;
+    
+    return workspaceClusterStore.remove(clusterItem);
+  }
+
+  @autobind()
+  settings() {
```

Comment:
This sounds like a getter, maybe rename to `gotoSettings` or `navigateToSettings` or `showSettings`?